ui/js/component/tipLink/view.jsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { Modal } from "component/modal";
+import Link from "component/link";
+import { FormField } from "component/form";
+
+class TipLink extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showTipBox: false,
+      feeAmount: "1.00",
+    };
+  }
+
+  handleTipPublisherButtonClicked() {
+    this.setState({
+      showTipBox: true,
+    });
+  }
+
+  handleTipButtonClicked() {
+    let address = this.props.claim.address;
+    let amount = this.state.feeAmount;
+
+    this.props.setAddress(address);
+    this.props.setAmount(amount);
+    this.props.sendToAddress();
+
+    this.setState({
+      showTipBox: false,
+    });
+  }
+
+  handleTipCancelButtonClicked() {
+    this.setState({
+      showTipBox: false,
+    });
+  }
+
+  handleFeeAmountChange(event) {
+    this.setState({
+      feeAmount: event.target.value,
+    });
+  }
+
+  render() {
+    const { modal, closeModal } = this.props;
+    const { showTipBox } = this.state;
+
+    let tipLink = (
+      <Link
+        label={__("Tip Publisher")}
+        button="text"
+        icon="icon-gift"
+        onClick={this.handleTipPublisherButtonClicked.bind(this)}
+      />
+    );
+
+    let tipBox = (
+      <span>
+        <FormField
```

Comment:
An LBC/USD input now appears:
- Here
- On Publish
- On Settings

It should probably just be turned into it's own component.